guides/micronaut-k8s/micronaut-k8s.adoc

```diff
@@ -0,0 +1,898 @@
+common:header-top.adoc[]
+
+== Getting Started
+
+In this guide, we will create three microservices, build containerized versions and deploy them with https://kubernetes.io/[Kubernetes]. We will use Kubernetes Service discovery and Distributed configuration to wire up our microservices.
+
+____
+Kubernetes is a portable, extensible, open source platform for managing containerized workloads and services, that facilitates both declarative configuration and automation. It has a large, rapidly growing ecosystem. Kubernetes services, support, and tools are widely available.
+____
+
+You will discover how the Micronaut framework eases Kubernetes integration.
+
+external:micronaut-k8s/requirements.adoc[]
+
+common:completesolution.adoc[]
+
+== Writing the Apps
+
+Let's describe the microservices you will build through the guide.
+
+* `users` - This microservice contains all customers data that can place orders on items, also a new customer can be created. Microservice requires Basic authentication to access it.
```

Comment:
```suggestion
* `users` - This microservice contains customers data that can place orders on items, also a new customer can be created. Microservice requires Basic authentication to access it.
```
This would be better renamed as `customers`. Why is authentication required? It seems to add unnecessary complexity to the guide. I don't think any of the other guides include it.